src/together/cli/api/finetune.py

```diff
@@ -1,15 +1,17 @@
 from __future__ import annotations
 
 import json
+from datetime import datetime
 from textwrap import wrap
 
 import click
 from click.core import ParameterSource  # type: ignore[attr-defined]
+from rich import print as rich_print
```

Comment:
in one place you import as rich_print, in other as just print
let's import always as "print"? I believe it's 1to1 replacement for print so should be fine

Reply:
I'm not a big fan of replacing builtins, as it makes understanding the code more difficult (you don't know which function is called). Rich suggests [in their docs](https://rich.readthedocs.io/en/stable/introduction.html#quick-start) to use `rprint`, so I will switch to that